### DISCUSSION TITLE
Expected server HTML to contain a matching <main> in <body>. is shown when rendering pages in Next.js 13.1-/app directory. How Can I fix It?

### DISCUSSION THREAD

**Discussion by pantcr7** [vercel/next.js#45210] [Help]
![Screenshot 2023-01-24 110122](https://user-images.githubusercontent.com/77143354/214219139-2458ffa2-0779-4ba0-baf1-c0b6374be85b.jpg)
 I got the above error while running dev cmd but not after building the application on my browsers. This issue was not found in other users' desktop browsers.

**Comment by guitheengineer** [comment]
### Why you don't see it after building

It's because this error isn't shown in production.

### When this error happens

When there's a mismatch between what's being rendered on the server and what's being rendered on the client. There's a lot of reasons this can happen, so is hard to guess without reading code, but a common one that is probably happening here is doing some conditional rendering with a variable that has different values between server and the client.
